app/src/main/java/com/kickstarter/ui/adapters/CommentsAdapter.kt

```diff
@@ -4,27 +4,52 @@ import android.view.LayoutInflater
 import android.view.ViewGroup
 import androidx.annotation.LayoutRes
 import com.kickstarter.R
+import com.kickstarter.databinding.EmptyViewBinding
 import com.kickstarter.databinding.ItemCommentCardBinding
+import com.kickstarter.databinding.ItemErrorPaginationBinding
 import com.kickstarter.ui.data.CommentCardData
 import com.kickstarter.ui.viewholders.CommentCardViewHolder
 import com.kickstarter.ui.viewholders.EmptyCommentsViewHolder
+import com.kickstarter.ui.viewholders.EmptyViewHolder
 import com.kickstarter.ui.viewholders.KSViewHolder
 
 class CommentsAdapter(private val delegate: Delegate) : KSListAdapter() {
     interface Delegate : EmptyCommentsViewHolder.Delegate, CommentCardViewHolder.Delegate
 
-    @LayoutRes
-    override fun layout(sectionRow: SectionRow): Int {
-        return R.layout.item_comment_card
+    init {
```

Comment:
@sunday-okpoluaefe this is gonna generate conflicts with your current work in progress, let me know if any doubt we can pair program for solving it